torch/_inductor/runtime/triton_heuristics.py

```diff
@@ -499,11 +499,14 @@ def _precompile_config(self, cfg: Config, warm_cache_only: bool):
                     so we use self.fn.constexprs instead.
             3. It isn't in the compile_meta signature
         """
-        none_args = set(compile_meta["constants"].keys())
         known_constants = {
             arg for i, arg in enumerate(self.fn.arg_names) if i in self.fn.constexprs
         }
-        none_args = none_args.difference(known_constants)
+        none_args = {
+            k
+            for k, v in compile_meta["constants"].items()
+            if v is None and k not in known_constants
```

Comment:
`v is None` doesn't seem to be a part of the previous code. Is this intended?

Reply:
Yes; before we didn't check for None values here. I added this while debugging the issue, but think it should probably stay as an added check (the arg we are filtering out should be None).